lib/context-base.js

```diff
@@ -29,3 +39,7 @@ ContextBase.prototype.getScope = function() {
     method.ctor ||
     method.sharedMethod && method.sharedMethod.ctor;
 };
+
+ContextBase.prototype.setReturnArgByName = function(name, value) {
+  // no-op
+};
```

Comment:
There should be end-of-line character at the end of the file. GitHub UI is smart to not display it. If this character was missing, GitHub would display a red symbol.